### ISSUE TITLE
[Boss] Leotheras the Blind

### ISSUE THREAD

**Issue by neheri** [SunwellWoW/Sunwell-TBC-Bugtracker#4136]
Description: Leotheras' demons are immune to shadow bolt reflect. 

How it works: Leotheras' demons are immune to shadow bolt reflect. 

How it should work: Leotheras' demons should not be immune to reflect.

Source (you should point out proofs of your report, please give us some source): 
http://wowwiki.wikia.com/wiki/Leotheras_the_Blind

When the inner demon is there, spell reflect the demon's shadow bolt. Wear a mix of DPS and tanking gear for this fight.

**Comment by neheri** [comment]
I consider this urgent because at the moment when you get inner demon as a prot war it's impossible to kill him.

**Comment by mchayes** [comment]
Going to pile on Leotheras here. 

He is showing weird behaviour in his execute phase regarding Whirlwinds, I can't find other data on it in the github. 
Hope it makes sense - boils down to him casting double WW's in execute phase at a random cooldown inbetween. If the gap is big enough the WW can be cast permanently during execute.

![](https://i.gyazo.com/361538c1a6ad56678953a6b5a7a80626.png)

**Comment by neheri** [comment]
@mchayes I agree. His whirlwind seems to be very odd to what I've experienced on other servers. 

- I was tanking him and he chased me for very long, sometimes for 50% of more of whirlwing time.
- During execute phase he entered demon phase... We had 2 demons at the same time. He should stay in elf phase until his death.

**Comment by mchayes** [comment]
Indeed, that made it considerably easier to kill! (regarding the demon shifting issue)

**Comment by neheri** [comment]
Please try to fix his shadow bolt reflect immunity quickly, it should be easy fix. When I used spell reflect when he casted on me, I didn't took damage so the bolt was reflected, however there was no damage taken by him, no message "immune" or "resist" and it happened on multiple reflect.

On the other servers it was super easy for prot war, 2-3 reflects and he was dead. At the moment, unless you use 30 min CD recklessness, you just can't kill him as a prot war.

**Comment by Macnjacks** [comment]
I also noticed that that the rend that is applied during whirlwind ticks rather high (maybe by design?). If a cloth player gets 2 rends they are basically dead. I know there is a away to avoid it, but with how Leo currently almost locks onto a player for a long period of time it's hard for 1 person to continue to run without getting hit twice. We even had a phase where he whirlwinded the tank the entire duration and got 8 rends and died.

**Comment by neheri** [comment]
In addition to everything stated above, on 15% when he splits he seems to be bugged as well.

First of all he keeps running while kneeling and seems to have preserved aggro from previous form (elf or demon). He killed our warlock tanking demon and it was impossible to regain aggro on him, despite several shield slams and heroic stikes.

Second, he seems to stay kneeling for too long. It should be few seconds after which he returns to being an elf and starting whirlwing. It may have been caused by a fact, that we entered 15% while he was in his demon form.

**Comment by neheri** [comment]
Update after killing him yesterday:

What seems to be fixed:
- after 15% he seems to stay in his elf form, does not transform into demon
- you can do nothing to other people's Inner demons

What seems to be bugged:
- after split at 15% his elf form whirlwind/melee timers are bad. He stayed in whirlwind for approximately 30 seconds, which resulted in killing a lot of people and stacking tanks for 8+ dots. 
- after his extended whirlwing he stayed in meele for only like 5 seconds and returned to extended whirlwind phase
- at 15% when he splits, he is moving while he's kneeling. He should be stationary until he stands up. He should start WW immediately after spliting. Yesterday he retained aggro from his demon form and went after our warlock tank
- his Insidious Whisper (demons summoning) was interrupted once for some reason. I am not sure if that was interrupt from one of our team members or he interrupted himself by casting fireball. It should never be interrupted

What we could not test:
- if inner demons' shadow bolts are reflectable (none of our warriors received inner demon yesterady)

Biggest problem: his WW after 15% is broken, which makes him a lot harder

**Comment by mchayes** [comment]
Leotheras is looking good at the moment, however his Chaos Blast is interruptable. I can not reproduce this in the combatlog since it is a hidden cast of sorts. No clue how this is going down. 

Either way my shaman rotation of FlameShock>EarthShock produced an accidental interrupt on him. 

Edit, I couldn't find it at first but then I realised theres something fucked up with combatlog, not sure if that's just me. Whenever you interrupt on this server the combatlog changes the interrupted spell to the spell you interrupt with. 
i.e. the interrupted chaos blast changes to this;
![](https://i.gyazo.com/438b9436c2edfaad6d5a10d71977e689.png)

**Comment by darknessangelo** [comment]
Summoned deamons in deamon phase are not visually linked to the players as you can see at 5:00 https://www.youtube.com/watch?v=HnDQi2RCkSE&t=300s
